scripts/create_static.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+
+"""
+Compile static assets via script
+
+Usage: python ./scripts/create_static.py [OPTIONS]
+
+"""
+
+import argparse
+import subprocess
+import main
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-d',
+                    help='Run from within python docker container.',
+                    action='store_true')
+args = parser.parse_args()
+
+if args.d:
+    commands = ['echo "Compiling static assets..."',
+                'docker-compose run --rm server -c' +
+                    '''"import subprocess;subprocess.call('./scripts/main.py')"''',
+                'echo "Finished compiling static assets."']
+    for command in commands:
+        subprocess.call(command, shell=True)
+else:
+    main.template_index()
```

Comment:
Would it be possible and desirable to have this command print some output when it runs successfully. I ran it locally and it exited successfully but there wasn't any output indicating what happened:

```
kinnes@allegheny:~/dvlp/GeositeFramework     (jf/port-region-json-templating *)$ python scripts/create_static.py
kinnes@allegheny:~/dvlp/GeositeFramework     (jf/port-region-json-templating *)$ echo $?
0
```

Reply:
Good point. I'll standardize logging!